recipes/meta/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "meta" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "a35dd5bc3ccffaaf83aa488233467ae85dc5374aea3068c34a6dc85ea29e27a9" %}
+{% set build_variant = "osmesa" if linux else "qt" %}
```

Comment:
you need to add a build variant file like they do in:
https://github.com/conda-forge/vtk-feedstock/blob/46b39d5b329e5a45a64e196a65cbb001f1fd2247/recipe/conda_build_config.yaml#L3

`conda_build_config.yaml` in this directory with the contetn:

```
build_variant:
  - "qt"
  - "osmesa"  # [linux64]
```

Reply:
I created `conda_build_config.yaml` and added build variant.